
Lmod is automatically replacing "intel/2020.1.217" with "gcc/9.3.0".


Due to MODULEPATH changes, the following have been reloaded:
  1) libfabric/1.10.1     2) openmpi/4.0.3     3) ucx/1.8.0

Traceback (most recent call last):
  File "tools/train.py", line 128, in <module>
    main(args)
  File "tools/train.py", line 101, in main
    cfg, device, args = check_and_init(args)
  File "tools/train.py", line 83, in check_and_init
    cfg = Config.fromfile(args.conf_file)
  File "/gpfs/scratch/rayen/NEU_DET_YOLOV63/yolov6/utils/config.py", line 62, in fromfile
    cfg_dict, cfg_text = Config._file2dict(filename)
  File "/gpfs/scratch/rayen/NEU_DET_YOLOV63/yolov6/utils/config.py", line 53, in _file2dict
    raise IOError('Only .py type are supported now!')
OSError: Only .py type are supported now!
Namespace(batch_size=32, conf_thres=0.03, config_file='', data='aug-fs-metatrain/neu_det.yaml', device='0', do_coco_metric=True, do_pr_metric=True, eval_config_file='./configs/experiment/eval_640_repro.py', half=False, img_size=320, infer_on_rect=True, iou_thres=0.65, name='eval_train_6l6-fs-32-320', plot_confusion_matrix=True, plot_curve=True, reproduce_640_eval=False, save_dir='results', shrink_size=0, task='val', verbose=True, weights='results/train_6l6-fs-32-320/weights/best_ckpt.pt')
checkpoint best_ckpt.pt not exist, try to downloaded it from github.
Traceback (most recent call last):
  File "tools/eval.py", line 156, in <module>
    main(args)
  File "tools/eval.py", line 151, in main
    run(**vars(args))
  File "/gpfs/scratch/rayen/NEU_DET_YOLOV63/ENV/lib/python3.8/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "tools/eval.py", line 140, in run
    model = val.init_model(model, weights, task)
  File "/gpfs/scratch/rayen/NEU_DET_YOLOV63/yolov6/core/evaler.py", line 64, in init_model
    download_ckpt(weights)
  File "/gpfs/scratch/rayen/NEU_DET_YOLOV63/yolov6/utils/general.py", line 96, in download_ckpt
    assert r.status_code == 200, "Unable to download checkpoints, manually download it"
AssertionError: Unable to download checkpoints, manually download it
